static/app/utils/regions/index.tsx

```diff
@@ -0,0 +1,91 @@
+import ConfigStore from 'sentry/stores/configStore';
+import {Organization, Region} from 'sentry/types';
+
+export enum RegionDisplayName {
+  US = 'United States of America (US)',
+  DE = 'European Union (EU)',
```

Comment:
```suggestion
  US = t('United States of America (US)'),
  DE = t('European Union (EU)'),
```

needs translations

Reply:
This ended up being a little more involved than I thought. Turns out Typescript gets fussy about using dynamic values in enums. I also tried adding the translation to the `getRegionDisplayName` function, but was getting a similar linting error: 

`Dynamic value interpolation cannot be used in translation functions. Use a parameterized string literal instead  sentry/no-dynamic-translations`

Any suggestions for cleaning this up?

Reply:
I would just not make it an enum, just use a Record<>